### ISSUE TITLE
Discord: MFA Code does not work

### ISSUE THREAD

**Issue by MP36ctrl** [miranda-ng/miranda-ng#4787]
[netlog.txt](https://github.com/user-attachments/files/17811001/netlog.txt)
MFA Code is wrong apparently even though it is indeed correct,
When entered, it forces all regular discord clients to go offline for some reason,
It seems like it needs a version update maybe?

**Comment by georgehazan** [comment]
@MP36ctrl did it work, ever?

**Comment by MP36ctrl** [comment]
It was working until 4 days ago, I tried backup codes and it is doing the same thing

**Comment by dartraiden** [comment]
Can you try to login via official website?

**Comment by MP36ctrl** [comment]
Yes, on regular website it works fine, its only with miranda it causes everything to logout, although it is not forcing me to reset my password like it used to every few weeks

**Comment by MP36ctrl** [comment]
It is also worth noting that the new discord proto update with api v10 is still not working for me

**Comment by dartraiden** [comment]
Of course, because the official app still uses v9. The problem is something completely different.

In my tests, as soon as Miranda receives an error from the server about an invalid code, I immediately lose the ability to enter the code even in the browser for a while.

**Comment by MP36ctrl** [comment]
For me it forces all my clients to log out, though i can login without waiting, before it used to force me to reset my password when it felt like it, but now its instantly causing issues, I have other 3rd party clients without this issue because they use token login which is good for situations like this, could there be a way to add an option for token login to miranda to avoid this issue?

**Comment by MP36ctrl** [comment]
is there a way i can test token login from dbeditorPP or something?

**Comment by MP36ctrl** [comment]
@georgehazan @dartraiden any update on this situation?

**Comment by georgehazan** [comment]
nope

**Comment by dartraiden** [comment]
@georgehazan I suggest to return Miranda to v9 API.
v10 did not help, official web-client still uses v9

**Comment by dartraiden** [comment]
Also I successfully capture entire login process in Fiddler,
I think we missing smth, can you look on it?

**Comment by MP36ctrl** [comment]
Should i try login you are saying?

**Comment by georgehazan** [comment]
> @georgehazan I suggest to return Miranda to v9 API.

It doesn't work either.
